apps/sertra/components/mod-selector/modal.module.css

```diff
@@ -25,66 +25,64 @@
 }
 
 .header {
-  background-color: var(--bg-light);
   display: flex;
   flex-direction: column;
-  padding: 24px 32px 16px;
+  padding: 1.5rem 2rem 1rem;
+  background-color: var(--bg-light);
 }
 
 .headerTitle {
   display: flex;
-  flex-direction: row;
+  flex-direction: row-reverse;
   align-items: center;
   justify-content: space-between;
+  margin-bottom: 1rem;
+}
+
+.headerTitle:last-child {
```

Comment:
Consider using a more directed approach for reusability. This works now, but might introduce problems if the header content changes.

Reply:
On the other hand, YAGNI. And if we need to change this, changing it to work another way around won't take practically any time.